### PR TITLE
2.4.5 support

### PR DESCRIPTION
**Pull request by npluimers**

### CODE REVIEW

**Review by legweilo — APPROVED** [review]

**Review by CptCharlesG — APPROVED** [review]

**Review by verhaeghep — APPROVED** [review]

File: view/frontend/web/js/price-utils.js [hunk 1]
```diff
@@ -26,6 +26,27 @@ define([
...
+
+        format = _.extend(globalPriceFormat, format);
+        precision = isNaN(format.requiredPrecision = Math.abs(format.requiredPrecision)) ? 2 : format.requiredPrecision;
+        pattern = format.pattern || '%s';
+        locale = window.LOCALE || 'en-US';
+        if (isShowSign === undefined || isShowSign === true) {
+            s = amount < 0 ? '-' : isShowSign ? '+' : '';
+        } else if (isShowSign === false) {
+            s = '';
+        }
```
Comment (lillik) [inline view/frontend/web/js/price-utils.js]: s = '';
        if (isShowSign === undefined || isShowSign === true) {
            s = amount < 0 ? '-' : isShowSign ? '+' : '';
        }

**Review by lillik — CHANGES_REQUESTED** [review]

File: view/frontend/web/js/price-utils.js (hunk 1 above)
Comment (lillik) [reply view/frontend/web/js/price-utils.js]: Please, try this and I'll merge your code. Thanks all for yours implication in order to make better this module. @npluimers @pfortin-expertime @CptCharlesG @verhaeghep

**Comment by verhaeghep** [comment]
The function formatPriceLocale is copied from original Magento2 source code. Don't see any reason to change this.
https://github.com/magento/magento2/blob/2.4.5-p1/app/code/Magento/Catalog/view/base/web/js/price-utils.js

**Comment by npluimers** [comment]
While the function is indeed from magento 2 source, I have made the change suggested 👍

**Comment by WaPoNe** [comment]
@lillik could you accept this PR, please?
Changes are made by @npluimers 

I think it will be the best solution to create a new tag (2.0.0 ?) putting in also this accepted PR https://github.com/lillik/magento2-price-decimal/pull/73 (it is not in 1.0.8 tag)

**Review by WaPoNe — CHANGES_REQUESTED** [review]
Change

`return round($price, $this->getPricePrecision());`

to

`return round($price, (int) $this->getPricePrecision());`

to avoid exception.
See https://github.com/lillik/magento2-price-decimal/pull/66/files

**Comment by lillik** [comment]
Need to fix code conflicts.

**Comment by WaPoNe** [comment]
@npluimers could you add code I suggested and fix code conflicts?

Thank you.

**Comment by verhaeghep** [comment]
Why does it take so long to fix this?

**Comment by legweilo** [comment]
@verhaeghep  you can use my fork for the time being : "pfortin-expertime/magento2-price-decimal": "1.0.9"

But I don't want to steal the work from @lillik so as soon we will have a new version compatible with 2.4 I'll remove it.

**Comment by kasperth** [comment]
@lillik Any progress on this? Appreciate the good work!

**Comment by bertholdkremser** [comment]
@lillik Any progress on this? Would love to see this fixed. Appreciate the good work!

**Comment by lillik** [comment]
@kasperth @bertholdkremser I appreciate your work. The PR was merged.

**Pull request merged**